cluster-up/cluster/kind-k8s-1.17.0-ipv6/README.md

```diff
@@ -20,6 +20,17 @@ cluster is brought up with ipv6 support but without flannel or multi nic support
     if needed, docker can be tested with:
     `docker run --rm busybox ip a`  
     and make sure you get an ipv6 address  
+1. External IPv6 connectivity:  
+    Enable IPv6 NAT:
+    ```console
+    # ip6tables -t nat -A POSTROUTING -s 2001:db8:1::/64 -j MASQUERADE
```

Comment:
Please mention that the CIDR should be set is the one that configured on the CRI (on docker it is configured at /etc/docker/daemon.json)